src/main/java/com/ufcg/sad/models/opcao/Opcao.java

```diff
@@ -2,16 +2,11 @@
 
 import static com.ufcg.sad.models.util.Utils.TAMANHO_MAX_STRING;
 
-import javax.persistence.Column;
-import javax.persistence.Entity;
-import javax.persistence.FetchType;
-import javax.persistence.GeneratedValue;
-import javax.persistence.GenerationType;
-import javax.persistence.Id;
-import javax.persistence.JoinColumn;
-import javax.persistence.ManyToOne;
+import javax.persistence.*;
```

Comment:
Esses wildcards não são legais. É melhor retirar e apenas importar aquilo que for usar.

Reply:
Reforço a observação de arthur